### PR TITLE
feat: added windows support via docker linux containers and fixed

### PR DESCRIPTION
**Pull request by alvint91**
networking to actually use llamanator

- Added some extra env variables specifically for windows

### CODE REVIEW

File: deploy/llamanator-bash/.env.example [hunk 1]
```diff
@@ -9,6 +9,7 @@ ENABLE_OLLAMACPU=true
 ENABLE_OLLAMAGPU=false
 # If you want to automatically download llama2, llama3, mistral, nomic-embed, codellama and phi3 models, set this to true. This will take some time to download.
 ENABLE_OLLAMA_BASE_MODELS=true
+OLLAMA_MODELS="llama2 mistral nomic-embed-text codellama llama3 phi3"
```
Comment (alvint91) [inline deploy/llamanator-bash/.env.example:12]: Did not actually refactor the linux/mac to use this.

**Comment by alvint91** [comment]
Was able to get this to work on my windows PC using docker desktop targeting linux containers.

image

**Comment by mjtechguy** [comment]
Nice! Will review later today and test.

**Comment by alvint91** [comment]
Am thinking we should not deal with this yet otherwise we need to keep porting the shell script changes to windows. Lets create a windows one if we actually need it and we have a more polished product.

**Pull request closed without merging**